pages/voting-guides/index.tsx

```diff
@@ -0,0 +1,149 @@
+import { useState, useEffect } from "react";
+import { NextPage } from "next";
+import Head from "next/head";
+import { 
+  Scalars,
+  useUpsertVotingGuideMutation,
+  useVotingGuidesByUserIdQuery,
+  VotingGuideResult } from "generated";
+import { Layout, Avatar, FlagSection, Button } from "components";
+import { useAuth } from "hooks/useAuth";
+import styles from "./VotingGuides.module.scss";
+
+
+const VotingGuideCard = ({ guide }:{ guide: VotingGuideResult }) => {
+  const {user} = guide;
+  const {firstName, lastName, avatarUrl, username} = user;
```

Comment:
**nitpick**: this should have been fixed by eslint but we should be using spaces between brackets

Reply:
Was thinking we should add auto new line, spaces around operators and brackets and potentially throw warnings if anonymous inline functions are used.

Reply:
that is good stuff, i'll add those lint rules now.  feel free to add any lint rules you like as well!

Reply:
I forgot to format using prettier, my bad!